frontend/src/common/backend.ts

```diff
@@ -166,7 +167,90 @@ export type VersionDto = {
 export type AuditEventDto = {
   id: number;
   timestamp: Date;
-  type: string;
+  type: 'CREATE_VAULT' | 'UNLOCK_VAULT' | 'UPDATE_VAULT_MEMBERSHIP';
+}
+
+export type CreateVaultEventDto = AuditEventDto & {
+  userId: string;
+  vaultId: string;
+}
+
+export type UnlockVaultEventDto = AuditEventDto & {
+  userId: string;
+  vaultId: string;
+  deviceId: string;
+  result: 'SUCCESS' | 'UNAUTHORIZED';
+}
+
+export type UpdateVaultMembershipEventDto = AuditEventDto & {
+  userId: string;
+  vaultId: string;
+  authorityId: string;
+  operation: 'ADD' | 'REMOVE';
+}
+
+/* AuditLogEntityCache */
+
+export class AuditLogEntityCache {
```

Comment:
this should get an `invalidate` method, that is called upon click on "refresh" in the audit log UI

Reply:
Fixed in 6ab57deca6f103259b5f2cfde6738c8957d52159